### ISSUE TITLE
E1101 false positive for abstract properties

### ISSUE THREAD

**Issue by stmlange** [PyCQA/pylint#3231]
Hello,
this seems to be a rather minor thing (I haven't found any similar issues and the problem is present in the preview release), but when using abstract methods along with properties pylint falsely claims a `E1101 (no-member)` 

### Steps to reproduce
1. construct an abstract-class with an abstract property
```python
from abc import ABCMeta, abstractmethod


class A(metaclass=ABCMeta):
    def __init__(self):
        pass

    @property
    @abstractmethod
    def a(self):
        pass

    @classmethod
    def some_method(cls):
        return cls.a.issubset({2, 3})


class B(A):
    a = {1, 2, 3}


b = B()
print("some_method: %s" % b.some_method())
```

2. run pylint (e.g. `python3 -m pylint foo.py -E`)

### Current behavior
Pylint falsely assumes that `a` is a method and not a `property`:
```
python3 -m pylint foo.py -E
************* Module foo
foo.py:15:15: E1101: Method 'a' has no 'issubset' member (no-member)
```

### Expected behavior
pylint does not complain ;-)

### pylint --version output
```
pylint 2.4.3
astroid 2.3.3
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:21:23)
```

**Comment by PCManticore** [comment]
Thank you for creating an issue, I can reproduce the bug as well.

**Comment by Ronkiro** [comment]
Any workarounds for this?
